install_config/aggregate_logging_sizing.adoc

```diff
@@ -0,0 +1,253 @@
+[[contributing-to-docs-templates-topic-template]]
```

Comment:
please give it another tag: install-config-aggregate-logging-sizing (for example)